samples/caf_sensor_manager/README.rst

```diff
@@ -41,21 +41,46 @@ Single-core configuration
 =========================
 
 For the MCU with multiple cores, the default configuration uses one core to simulate the sensor and the other core to process the sensor.
-These multicore MCUs can support a single-core configuration, where the sensor is simulated and processed on a single, selected core.
+The multicore MCUs can also support a single-core configuration, where the sensor is simulated and processed on a single, selected core.
 The configuration is placed in the :file:`boards/<board>_singlecore.conf` file.
 
-To use this configuration, specify the ``-DOVERLAY_CONFIG=boards/<board>_singlecore.conf`` parameter along with the build command when building the sample:
+To use this configuration, run the following command:
 
-.. code-block:: console
+   .. tabs::
 
-   west build -b nrf5340dk_nrf5340_cpuapp -- -DOVERLAY_CONFIG=boards/nrf5340dk_nrf5340_cpuapp_nrf5340_singlecore.conf
+      .. group-tab:: nRF5340 DK
```

Comment:
We could remove the tab because the tab was introduced for different DKs representation.
But here we are just mentioning nRF5340 DK.